manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```
@@ -838,6 +838,36 @@ data:
     # Number of Tempo stacks with Jaeger UI enabled/disabled.
     - '{__name__="enabled:tempo_operator_tempostack_jaeger_ui:sum",enabled="true|false"}'
     #
+    # owners: (@tracing-team)
+    #
+    # Number of OpenTelemetry collectors used certain receiver type
```

(nit)  same for all other comments.
```suggestion
    # Number of OpenTelemetry collectors using certain receiver types.
```